zeppelin-zengine/src/main/java/com/teragrep/zep_01/interpreter/remote/RemoteAngularObject.java

```diff
@@ -46,9 +46,11 @@ public void set(Object o, boolean emitWeb, boolean emitRemoteProcess) {
 
     if (emitRemoteProcess) {
       // send updated value to remote interpreter
-      interpreterGroup.getRemoteInterpreterProcess().
-          updateRemoteAngularObject(
-              getName(), getNoteId(), getParagraphId(), o);
+      RemoteInterpreterProcess process = interpreterGroup.getRemoteInterpreterProcess();
```

Comment:
```suggestion
      final RemoteInterpreterProcess process = interpreterGroup.getRemoteInterpreterProcess();
```
`process` can be made final

Reply:
fixed in commit [f7ace22](https://github.com/teragrep/zep_01/pull/244/commits/f7ace22970fe654d84285babc5856784949324d6)

Reply:
`process` is now final